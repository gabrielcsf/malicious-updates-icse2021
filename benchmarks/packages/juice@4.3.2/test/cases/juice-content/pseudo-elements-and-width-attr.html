<html><body>
<style>
a {
  text-decoration: underline;
}

a:before,
a:after {
  content: "a";
}

* a:before {
  content: 'Â®\+';
}

a:after,
a::after {
  font-weight: bold;
}

a:first-child {
  color: blue;
}

b:after {
 content: " ";
}

b:after {
 content: "b";
 color: green;
}

b :last-child {
  color: red;
}
</style>
<a href="#">Test</a>
<b></b>
<b><span></span></b>
</body></html>
