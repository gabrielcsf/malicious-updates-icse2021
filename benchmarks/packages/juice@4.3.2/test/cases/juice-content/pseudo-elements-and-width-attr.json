{
  "url": "./",
  "removeStyleTags": true,
  "inlinePseudoElements": true,
  "applyWidthAttributes": true
}
