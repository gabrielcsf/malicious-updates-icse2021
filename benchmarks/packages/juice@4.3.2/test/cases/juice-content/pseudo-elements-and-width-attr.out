<html><body>

<a href="#" style="text-decoration: underline; color: blue;"><span>Â®+</span>Test<span style="font-weight: bold;">a</span></a>
<b><span style="color: green;">b</span></b>
<b><span style="color: red;"></span><span style="color: green;">b</span></b>
</body></html>
